.travis.yml

```diff
@@ -1,30 +1,23 @@
 sudo: false
 language: python
-python:
-    - "2.7"
-    - "3.4"
```

Comment:
> Tox controls the Python versions tested, so we only need one Python
version specified at the Travis level

This isn't entirely true. I think what you mean is that tox controls the virtualenv for each Python. The machine still needs each version of Python installed otherwise tox won't be able to find an executable for it. I think the tests were working before (with the duplicate entry typo) because travis makes some versions available by default. I'd rather see those versions declared explicitly for when travis removes old versions in the defaults.